examples/jest/package.json

```diff
@@ -8,7 +8,8 @@
     "url": "https://github.com/mtkennerly/kelonio.git"
   },
   "scripts": {
-    "test": "npx jest --verbose --silent --runInBand"
+    "test": "npx jest --verbose --silent --runInBand",
+    "test:keepStateAtStart": "KELONIO_KEEP_STATE_AT_START=true npm test"
```

Comment:
This is not necessary. You already override the environment variable in the tests.